Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/function/EditFunctionSignatureDialog.java

```diff
@@ -322,10 +322,6 @@ protected boolean applyChanges() throws CancelledException {
 		// create the command
 		Command command = createCommand();
 
-		if (command == null) {
-			return false;
-		}
-
 		// run the command
 		if (!getTool().execute(command, getProgram())) {
 			setStatusText(command.getStatusMsg());
```

Comment:
The removal here will result in a NullPointerException in ToolTaskManager#applyCommand if command is null.

Reply:
Hm I'm not sure what you mean. `command == null` is always `false`, as `command` is derived from `createCommand()`, which can not return null, because in this case it would have thrown an NPE for example in https://github.com/NationalSecurityAgency/ghidra/blob/master/Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/function/EditFunctionSignatureDialog.java#L431. If a NPE is thrown, this will not be reached.